DataFormats/simulation/include/SimulationDataFormat/Stack.h

```diff
@@ -145,6 +145,10 @@ class Stack : public FairGenericStack
 
     virtual void Print(Int_t iVerbose = 0) const;
 
+    /// Output to screen (derived from base class)
+    /// \param option: 0=events summary, non0=track info
+    virtual void Print(Option_t* option = 0) const;
+
```

Comment:
C++11: prefer marking with override. The pull request will probably fail in future if things like this are not done.